airbyte-integrations/singer/stripe_abprotocol/source/source_stripe_singer/source.py

```diff
@@ -8,10 +8,13 @@ def __init__(self):
         pass
 
     def check(self, logger, config_container) -> AirbyteCheckResponse:
-        json_config = config_container.rendered_config
-        r = requests.get('https://api.stripe.com/v1/customers', auth=(json_config['client_secret'], ''))
+        try:
+            json_config = config_container.rendered_config
+            r = requests.get('https://api.stripe.com/v1/customers', auth=(json_config['client_secret'], ''))
 
-        return AirbyteCheckResponse(r.status_code == 200, {})
+            return AirbyteConnectionStatus(status=Status.SUCCEEDED if (r.status_code == 200) else Status.FAILED)
```

Comment:
If it is not 200, it would be helpful to have a message

Reply:
Good idea, will change.